src/nimlsp.nim

```diff
@@ -32,23 +32,10 @@ type
 
 var nimpath = explicitSourcePath
 
-discard existsOrCreateDir(storage)
```

Comment:
Please don't remove this statement, the rest of the project also depends on this folder.